src/module/apps/svelte-mixin.svelte.ts

```diff
@@ -0,0 +1,43 @@
+import * as svelte from "svelte";
+import type {
+    ApplicationRenderContext,
+    ApplicationRenderOptions,
+} from "types/foundry/client-esm/applications/_types.d.ts";
+import type ApplicationV2 from "types/foundry/client-esm/applications/api/application.d.ts";
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+function SvelteApplicationMixin<TBase extends AbstractConstructorOf<ApplicationV2>>(Base: TBase) {
+    abstract class SvelteApplication extends Base {
```

Comment:
Don't we have one of these? I ended up doing a bugfix to the replaceHTML() implementation that is somewhat similar to yours, if you wish to poach it, that maintains $state being a protected property.

Reply:
That is a vestige from me starting with the browser before the abc-picker was merged. I think the mixin location was moved multiple times and I forgot to update that-